Cargo.toml

```diff
@@ -17,10 +17,14 @@ path = "src/passthrough.rs"
 
 [dependencies]
 tokio = { version = "*", features = [ "full" ] }
+zeroconf = { version = "*", path = "../zeroconf-rs/zeroconf", optional = true }
```

Comment:
The only reason I specified a path for mDNS was because I fixed a bug in it. Is there a reason we aren't using zeroconf from crates.io?